packages/ui/src/PageLayout.tsx

```diff
@@ -7,7 +7,7 @@ type PageLayoutProps = {
 
 const gapMap = {
   16: 'gap-16',
-  32: 'gap-32',
+  32: 'sm:gap-32 gap-16',
```

Comment:
I can't find this back in the design .. ?

Reply:
Might be a design hiccup - Home, About and Projects have `gap-32` on all screens, but the rest of the pages have [`gap-16` on `sm` screen. ](https://www.figma.com/design/P67wEFE8S7ay0b7FErfqPE/FFDweb.org-Prototypes?node-id=5677-56322&t=5zpUvTGCKR8EJM6S-4)